cfme/services/myservice.py

```diff
@@ -250,6 +250,12 @@ def download_file(self, extension):
         view.download_choice.item_select("Download as " + extension)
         view.flash.assert_no_error()
 
+    def reconfigure_service(self):
+        view = navigate_to(self, 'Reconfigure')
+        view.submit_button.click()
+        # Request page is displayed so not asserting for view
+        view.flash.assert_no_error()
```

Comment:
**Required**
Add a TODO here to create view and assert against the redirect once Requests is modeled with widgetastic.  The flash widget here is inherited through several views.